ntoskrnl/se/token.c

```diff
@@ -83,6 +82,64 @@ static const INFORMATION_CLASS_INFO SeTokenInformationClass[] = {
 
 /* FUNCTIONS *****************************************************************/
 
+/**
+ * @brief
+ * Creates a lock for the token.
+ * 
+ * @param[in,out] Token
+ * A token which lock has to be created.
+ *
+ * @return
+ * STATUS_SUCCESS if the pool allocation and resource initialisation have
+ * completed successfully, otherwise STATUS_INSUFFICIENT_RESOURCES on a
+ * pool allocation failure.
+ *
+ */
+static
```

Comment:
```suggestion
static inline
```

Reply:
No

Reply:
Since these functions are short and used only once or twice they can be made inline.

Reply:
`static inline` does not have any difference to `static` in this case. Only wasting lines